cmd/main.go

```diff
@@ -67,32 +66,32 @@ type params struct {
 }
 
 func executeStringCmd(meth method, pth string, pars params) (string, error) {
-	req, err := request(meth, pth, pars)
+	res, _, err := request(meth, pth, pars)
 	if err != nil {
 		return "", err
 	}
-	defer req.Body.Close()
-	res, err := util.UnmarshalString(req.Body)
+	defer res.Body.Close()
+	body, err := util.UnmarshalString(res.Body)
 	if err != nil {
 		return "", err
 	}
-	return res, nil
+	return body, nil
 }
 
 func executeJsonCmd(meth method, pth string, pars params, target interface{}) (string, error) {
-	req, err := request(meth, pth, pars)
+	res, _, err := request(meth, pth, pars)
 	if err != nil {
 		return "", err
 	}
-	defer req.Body.Close()
```

Comment:
I did this renaming because it bit me once... wasted an hour or so dealing w/ the wrong `Body`, doh.